plugins/entando-plugin-jacms/src/test/java/org/entando/entando/plugins/jacms/web/content/ContentControllerIntegrationTest.java

```diff
@@ -249,6 +250,178 @@ public void testAddDeleteContent() throws Exception {
         }
     }
 
+    @Test
+    public void testUpdateContents() throws Exception {
```

Comment:
The test's name should be more adequate to the actual API method, for instance, regarding batch and status update for a single item.